avocado/core/runners/sysinfo.py

```diff
@@ -0,0 +1,298 @@
+import logging
+import multiprocessing
+import os
+import time
+import traceback
+from abc import ABC, abstractmethod
+
+from avocado.core import nrunner
+from avocado.core.runners.utils import messages
+from avocado.core.settings import settings
+from avocado.utils import astring, process, software_manager
+
+__all__ = ["PreSysInfo", "PostSysInfo", "SysinfoRunner", "RunnerApp", "main"]
```

Comment:
While I understand that there was a discussion about [this ](https://github.com/avocado-framework/avocado/pull/4760#discussion_r667897278), I don't think the intended outcome was to include an `__all__`.

It should only be used on modules that you intend for others to import public symbols from, and contain private symbols.  I believe we fail on the first check here, so I suggest this is simply dropped, for, among the reasons above, to keep consistency with all other modules.

Reply:
After revisiting all the classes here, yes I agree. Let's drop __all__ and keep the class names as it is.